### DISCUSSION TITLE
Add new language

### DISCUSSION THREAD

**Discussion by itsnotsaved** [home-assistant/frontend#9832] [Localization]
This is a request to add sinhala language to all home assistants projects.

Language: Sinhala
Name: සිංහල
Code: si

Projects:  https://developers.home-assistant.io/docs/translations
- [ ] Frontend
- [ ] Backend
- [ ] iOS
- [x] Android

Furthermore, i contribute as a translator / proofreader for eg: lineageos, joomla etc projects.
Email: R45XvezA@pm.me | Name: HelaBasa

@robbiet480 can you enable this language to below projects? but do you still maintain them? because i didn't see any details about them on translation docs

https://crowdin.com/project/home-assistant-companion-docs
https://crowdin.com/project/home-assistant-developer-docs

**Comment by itsnotsaved** [comment]
@bramkragten

**Comment by dshokouhi** [comment]
@itsnotsaved I have completed this for Android....for iOS file a separate request with that repo directly so they can add you.
